applications/GeoMechanicsApplication/tests/test_thermal_element/test_thermal_line_element/test_thermal_line_element_3D2N/MaterialParameters.json

```diff
@@ -0,0 +1,30 @@
+{
+  "properties": [{
+    "model_part_name"                      : "PorousDomain.filter",
+    "properties_id"                        : 1,
+    "Material" : {
+      "constitutive_law": {
+        "name"                             : "GeoThermalDispersion2DLaw" 
```

Comment:
For consistency, I would prefer to use `"GeoThermalDispersion3DLaw"` here (this is a 3D test, after all).